### PR TITLE
Re-render master with conda-smithy 1.3.2 [ci skip]

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes the re-rendering in PR ( https://github.com/conda-forge/tk-feedstock/pull/8 ) and undoes the reversion in PR ( https://github.com/conda-forge/tk-feedstock/pull/9 ).

### CODE REVIEW

**Comment by jakirkham** [comment]
Hopefully this helps, @jjhelmus.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Uncomment for Windows re-rendering only.
+#
+#matrix:
+#  - [[VC_VERSION, 9], [CONDA_PY, 27]]
+#  - [[VC_VERSION, 10], [CONDA_PY, 34]]
+#  - [[VC_VERSION, 14], [CONDA_PY, 35]]
```
Comment (jakirkham) [inline conda-forge.yml]: Basically my suggestion until we have something is re-render with this commented and commit all changes. Then uncomment this, re-render, and only commit `appveyor.yml` (possibly amending the last re-rendering commit). That should get us a little closer to a stable process for doing this.

**Comment by jjhelmus** [comment]
LGTM, not sure why AppVeyor is showing yellow when the build is green when I click for details.  Thanks @jakirkham

**Comment by jakirkham** [comment]
Sometimes the webhooks with AppVeyor don't update. Not sure why, but AppVeyor is going to start giving us more info, which may be helpful for chasing down problems like this one.

**Pull request merged**